cpp/ql/test/library-tests/dataflow/dataflow-tests/test.cpp

```diff
@@ -1046,6 +1046,14 @@ void memset_test(char* buf) { // $ ast-def=buf ir-def=*buf
 	sink(*buf); // $ ir MISSING: ast
 }
 
+void *realloc(void *, size_t);
+
+void test_realloc() {
+	int *src = indirect_source();
+	int *dest = (int*)realloc(src, sizeof(int));
+	sink(*dest); // $ ir, MISSING: ast
```

Comment:
FWIW, this is not really testing that `realloc` is a `dataflow` function. This test would also pass with `realloc` being a taint-function.

See the testcase I provided in the internal issue if you want to fix this 😄 I don't really mind, though.